low-level/framework/sspl_ll_d

```diff
@@ -74,6 +74,7 @@ from message_handlers.real_stor_actuator_msg_handler import RealStorActuatorMsgH
 # Message to send to HAlon upon critical thread errors
 from json_msgs.messages.actuators.thread_controller import ThreadControllerMsg
 
+# Creating Coverage instance for coverage report generation
```

Comment:
DO NOT EDIT: Marker comment to dynamically add code to initialize coverage obj for code coverage report generation

Reply:
addressed